pkg/client/list.go

```diff
@@ -61,7 +62,8 @@ func ListDrives(ctx context.Context, driveInterface clientset.DirectCSIDriveInte
 			LabelSelector: labelSelector,
 		}
 		for {
-			result, err := driveInterface.List(ctx, options)
+			directClient := GetUnversionedDirectCSIDriveClientset()
```

Comment:
we don't need this, we can make use of the `driveInterface` function argument here.